workflow/controller/operator.go

```diff
@@ -116,8 +117,10 @@ var (
 
 // maxOperationTime is the maximum time a workflow operation is allowed to run
 // for before requeuing the workflow onto the workqueue.
-const maxOperationTime = 10 * time.Second
-const defaultRequeueTime = maxOperationTime
+var (
+	maxOperationTime   = envutil.LookupEnvDurationOr("MAX_OPERATION_TIME", 30*time.Second)
```

Comment:
I am fine with 30s because one of our ML customers is going really big workflow with 2000 + dynamic steps. 30s will help this workflow parse the big JSON parameters.   It is configurable so if there is any backlog issue, we configure it back to 10s or 20s